src/test/java/io/confluent/connect/hdfs/HdfsSinkTaskWithSecureHDFSTest.java

```diff
@@ -54,6 +55,8 @@ public void testSinkTaskPut() throws Exception {
         sinkRecords.add(sinkRecord);
       }
     }
+
+    properties.remove(StorageCommonConfig.STORE_URL_CONFIG);
```

Comment:
wondering if we should set the value of STORE_URL_CONFIG prop in io.confluent.connect.hdfs.TestWithSecureMiniDFSCluster#createProps (that is where the url for HDFS_URL_CONFIG is being overridden).